ably/realtime_conn_spec_test.go

```diff
@@ -1861,7 +1861,7 @@ func TestRealtimeConn_RTN14c(t *testing.T) {
 		ably.WithKey("xxx:xxx"),
 		ably.WithRealtimeHost(host),
 		ably.WithTLSPort(p),
-		ably.WithRealtimeRequestTimeout(reqTimeout),
+		ably.WithHTTPOpenTimeout(reqTimeout),
```

Comment:
This test for RTN14c should still be testing `realtimeRequestTimeout`.

Reply:
I did that change because the test broke after replacing `RealtimeRequestTimeout` with `HTTPOpenTimeout` for the initial connection. I'm ok with reverting that though.

Reply:
Opened https://github.com/ably/ably-go/issues/315 to fix this.